pype/plugins/tvpaint/publish/extract_sequence_exr.py

```diff
@@ -0,0 +1,65 @@
+import os
+import json
+
+import pyblish.api
+import pype.api
+
+
+class ExtractSequenceEXR(pyblish.api.InstancePlugin):
+
+    # Offset to get after ExtractSequence plugin.
+    order = pyblish.api.ExtractorOrder + 0.1
+    label = "Extract Sequence EXR"
+    hosts = ["tvpaint"]
+    families = ["renderPass", "renderLayer"]
+    active = False
+
+    def process(self, instance):
+        ignore_names = ["thumbnail"]
+        ignore_extensions = ["mp4"]
+        old_representations = instance.data["representations"]
+        new_representations = []
+        for count, representation in enumerate(old_representations):
+            if representation["name"] in ignore_names:
+                continue
+
+            if representation["ext"] in ignore_extensions:
+                continue
+
+            self.log.info(
+                "Processeing representation: {}".format(
+                    json.dumps(representation, sort_keys=True, indent=4)
+                )
+            )
+
+            files = []
+            for f in representation["files"]:
+                f = os.path.join(representation["stagingDir"], f)
+                path, ext = os.path.splitext(f)
+                output_path = path + ".exr"
+                args = [
+                    "oiiotool", f,
```

Comment:
would't this be better to use 
```suggestion
                    oiio_tool_path = openpype.lib.get_oiio_tools_path()
```

Reply:
this is for pype2 still, It don't think we have that option back there

Reply:
> this is for pype2 still, It don't think we have that option back there

You are right. No such thing in 2.x.

Reply:
In default `pype-config` is environment variable `PYPE_OIIO_PATH` which is used on few places:
```
"PYPE_OIIO_PATH": {
      "windows": "{PYPE_SETUP_PATH}/vendor/bin/oiiotool/windows/oiiotool.exe",
      "linux": "{PYPE_SETUP_PATH}/vendor/bin/oiiotool/linux/oiiotool",
      "darwin": "{PYPE_SETUP_PATH}/vendor/bin/oiiotool/darwin/oiiotool"
    },
```

Reply:
Good point! Done.